### PR TITLE
fix: disable proxy buffering and cache

### PR DESCRIPTION
**Pull request by nityanandagohain**
charts PR https://github.com/SigNoz/charts/pull/277

### CODE REVIEW

**Review by prashant-shahi — APPROVED** [review]

**Comment by prashant-shahi** [comment]
redundant PR. Public charts PR is enough. We later sync it with private saas.

**Review by prashant-shahi — CHANGES_REQUESTED** [review]
@nityanandagohain can you update the same for Swarm and other test config under `query-service`?

**Review by prashant-shahi — DISMISSED** [review]
LGTM :shipit:

**Comment by srikanthccv** [comment]
Same as https://github.com/SigNoz/charts/pull/277#issuecomment-1687500977

**Review by prashant-shahi — DISMISSED** [review]

File: deploy/docker-swarm/common/nginx-config.conf [hunk 1]
```diff
@@ -28,6 +28,18 @@ server {
...
 
+    location ~ ^/api/(v1|v3)/logs/(tail|livetail){
+        proxy_pass http://query-service:8080;
+        proxy_http_version 1.1;
+
+        # connection will be closed if no data is read for 600s between successive read operations
+        proxy_read_timeout 600s;
+
+        # dont buffer the data send it directly to client.
+        proxy_buffering on;
```
Comment (srikanthccv) [inline deploy/docker-swarm/common/nginx-config.conf:39]: You want to turn this off, right?

**Review by prashant-shahi — APPROVED** [review]

**Comment by prashant-shahi** [comment]
@srikanthccv Feel free to merge the PR after review from your end.

**Review by srikanthccv — APPROVED** [review]
Using this chance to plug something. One of the review principles that I keep in mind, among others

_look at every line of code that you have been assigned to review ... but don’t scan over a human-written class, function, or block of code and assume that what’s inside of it is okay._

https://google.github.io/eng-practices/review/reviewer/looking-for.html#every-line

The whole guidelines are good https://google.github.io/eng-practices/.

**Pull request merged**